cropduster/standalone/models.py

```diff
@@ -54,17 +54,18 @@ class StandaloneImage(models.Model):
 
     objects = StandaloneImageManager()
 
-    md5 = models.CharField(max_length=32)
+    md5 = models.CharField(max_length=32, blank=True, default='')
     image = CropDusterField(sizes=[Size("crop")], upload_to='')
 
     class Meta:
         app_label = cropduster_settings.CROPDUSTER_APP_LABEL
         db_table = '%s_standaloneimage' % cropduster_settings.CROPDUSTER_DB_PREFIX
 
     def save(self, **kwargs):
-        if not self.md5:
+        if not self.md5 and self.image:
             md5_hash = hashlib.md5()
-            with default_open(self.image.path, mode='rb') as f:
+            with self.image.related_object.image as f:
+                f.open()
                 md5_hash.update(f.read())
-            self.md5 = md5_hash.digest()
+            self.md5 = md5_hash.hexdigest()
```

Comment:
why the change from `digest` to `hexdigest`?

Reply:
`.digest()` actually returns bytes, but what we want is the hexadecimal md5 so we can save it in a CharField. It's also what we use elsewhere in cropduster, e.g.:

https://github.com/theatlantic/django-cropduster/blob/c2168c36f34922b509a98e3dc667441799389916/cropduster/views/forms.py#L88